examples/remarketing/add_customer_match_user_list.py

```diff
@@ -175,22 +173,13 @@ def _add_users_to_customer_match_user_list(
         )
     )
 
-    if skip_polling:
-        _check_job_status(
-            client,
-            customer_id,
-            offline_user_data_job_resource_name,
-            user_list_resource_name,
-        )
-    else:
-        # Wait until the operation has finished.
-        print("Request to execute the added operations started.")
-        print("Waiting until operation completes...")
-        operation_response.result()
-        _print_customer_match_user_list_info(
-            client, customer_id, user_list_resource_name
-        )
-        # [END add_customer_match_user_list]
+    _check_job_status(
```

Comment:
Can you add a comment about this to match the golden? 
```py
# Offline user data jobs may take 6 hours or more to complete, so instead of waiting for the
# job to complete, retrieves and displays the job status once. If the job is completed
# successfully, prints information about the user list. Otherwise, prints the query to use
# to check the job again later.
```